components/script/dom/bindings/root.rs

```diff
@@ -520,9 +521,12 @@ impl<T> Clone for Dom<T> {
 
 impl<T> Clone for LayoutDom<'_, T> {
     #[inline]
+    #[allow(clippy::non_canonical_clone_impl)]
     fn clone(&self) -> Self {
         assert_in_layout();
-        LayoutDom { value: self.value }
+        {
+            *self
+        }
```

Comment:
Is `{ *self }` any different than just `*self`?

Reply:
@Loirooriol, it seems like there's no difference between the two implementations. I will remove the unnecessary curly braces that were reapplied in the last commit.